### PR TITLE
FDTN-301: Updated Lightbend conf

### PR DESCRIPTION
**Pull request by Yatharth0045**

### CODE REVIEW

File: api/conf/base.conf [hunk 1]
```diff
@@ -30,10 +30,27 @@ rollbar.token = ${?ROLLBAR_TOKEN}
...
+cinnamon.datadog {
+  statsd {
+    host = "datadog-cluster-agent.datadog"
+    port = 5005
+    frequency = 60s
+  }
+
+  report {
+    histogram = ["min", "max", "p98", "p99", "p999"]
+  }
```
Comment (gheine) [inline api/conf/base.conf:42]: let's leave out the histogram section. the D agent will add default expansions to histogram metrics already

File: api/conf/base.conf [hunk 2]
```diff
@@ -30,10 +30,27 @@ rollbar.token = ${?ROLLBAR_TOKEN}
 
 JWT_SALT = ${?JWT_SALT}
 
+cinnamon.datadog {
+  statsd {
+    host = "datadog-cluster-agent.datadog"
+    port = 5005
+    frequency = 60s
```
Comment (gheine) [inline api/conf/base.conf:37]: host should be the unix domain socket, like `/var/run/datadog/dsd.socket` or `unix:///var/run/datadog/dsd.socket` and port should be 0

File: api/conf/base.conf [hunk 3]
```diff
@@ -30,10 +30,27 @@ rollbar.token = ${?ROLLBAR_TOKEN}
...
+  }
+
+  report {
+    histogram = ["min", "max", "p98", "p99", "p999"]
+  }
+}
+
 cinnamon.chmetrics {
```
Comment (gheine) [inline api/conf/base.conf:45]: This section can be dropped. The `chmetrics` config is only for Coda Hale metrics which we're already submitting through lib-metrics.

File: api/conf/base.conf (hunk 2 above)
Comment (gheine) [reply api/conf/base.conf:37]: possibly the config you had in `chmetrics` below should go here:
```
    channel = unix-domain-socket
    unix-domain-socket {
      path = /var/run/datadog/dsd.socket
    }
```

Comment (Yatharth0045) [reply api/conf/base.conf]: @gheine 
While going the configuration again, I saw that `cinnamon.datadog` is now outdated and not being used anymore in latest version. For the latest version, the steps are configured overe here. https://developer.lightbend.com/docs/telemetry/current/plugins/datadog/datadog.html

Updated this PR accordingly

Comment (gheine) [reply api/conf/base.conf]: I see. Ok, sounds good, let's try this out!

**Review by gheine — APPROVED** [review]

**Pull request merged**